devel/lint.sh

```diff
@@ -214,11 +219,20 @@ EOF
 }
 
 
+function is-in-lintignore
+{
+    local filepath="$1"
+    [ "${LINT_FILTER_BYPASS}" -eq 0 ] && grep -q "^${filepath}\$" devel/lint.ignore && return 0
+    return 1
```

Comment:
One problem: `$filepath` has a leading `./`, e.g. `./devel/lint.sh`.  But the values in `devel/lint.ignore` are not expected to have the leading `./`.

Solution: strip the leading `./` from `$filepath` variable.  The following change is not perfect but perhaps good enough:
```diff
diff --git a/devel/lint.sh b/devel/lint.sh
index f39fa9c..7ca9876 100755
--- a/devel/lint.sh
+++ b/devel/lint.sh
@@ -221,7 +221,7 @@ EOF
 
 function is-in-lintignore
 {
-    local filepath="$1"
+    local filepath=${1/.\//}
     [ "${LINT_FILTER_BYPASS}" -eq 0 ] && grep -q "^${filepath}\$" devel/lint.ignore && return 0
     return 1
 }
```

Reply:
:+1:  @frasertweedale thanks a million mate, I change this.

Reply:
At the end I have used the expression `filepath="${filepath#./}"` that remove the indicated prefix if it matches.

Reply:
Yep, that's a better solution.  Merging now, thanks!